src/models/thread.ts

```diff
@@ -358,7 +358,15 @@ export class Thread extends ReadReceipt<EmittedEvents, EventHandlerMap> {
             try {
                 // if the thread has regular events, this will just load the last reply.
                 // if the thread is newly created, this will load the root event.
-                await this.client.paginateEventTimeline(this.liveTimeline, { backwards: true, limit: 1 });
+                if (this.replyCount === 0 && this.rootEvent) {
+                    this.timelineSet.addEventsToTimeline([this.rootEvent], true, this.liveTimeline, null);
+                    this.liveTimeline.setPaginationToken(null, Direction.Backward);
+                } else {
+                    await this.client.paginateEventTimeline(this.liveTimeline, {
+                        backwards: true,
+                        limit: Math.max(1, this.length),
+                    });
+                }
```

Comment:
What was wrong before that caused https://github.com/vector-im/element-web/issues/23934?

What changed so https://github.com/vector-im/element-web/issues/23934 now works?

If there is some tricky order of operations here, we should explain why/what, etc

Reply:
@justjanne ^

Reply:
@MadLittleMods I'm currently on vacation, I'll address it in January.